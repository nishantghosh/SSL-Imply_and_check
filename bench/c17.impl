# Some test implications

fault_site(10,1)
imply(1,1)
J_frontier
imply(3,1)
imply(16,1)
J_frontier
D_frontier
x-path
display_lines
