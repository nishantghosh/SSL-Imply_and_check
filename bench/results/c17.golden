Parsing ISCAS Format... done.
Parsing commands file... done.
J-Frontier
$

J-Frontier
16
$

D-Frontier
$

X-PATH
$

Circuit State
1: 1
10: DBAR
11: X
16: 1
19: X
2: X
22: D
23: X
3: 1
6: X
7: X
$

